### PR TITLE
Document reply action images and share buttons

### PR DESCRIPTION
**Pull request by alavers**
@mspensieri @jugarrit @lemieux

### CODE REVIEW

File: source/includes/rest/_core.md [hunk 1]
```diff
@@ -1417,12 +1430,65 @@ smooch.appUsers.sendMessage('c7f6e6d6c3a637261bd9656f', {
...
+    "role": "appMaker",
+    "actions": [{
+        "type": "reply",
+        "text": "Tacos",
+        "iconUrl": "http://example.org/taco.png"
+        "payload": "TACOS"
+    }, {
+        "type": "reply",
+        "text": "Burritos",
+        "iconUrl": "http://example.org/burrito.png"
```
Comment (lemieux) [inline source/includes/rest/_core.md]: So... is it `iconUrl` or `imageUrl`?

**Comment by jpjoyal** [comment]
👍

File: source/includes/rest/_core.md (hunk 1 above)
Comment (lemieux) [inline source/includes/rest/_core.md]: Everywhere else it's called `imageUrl` and in the examples it's `iconUrl`

Comment (alavers) [inline source/includes/rest/_core.md]: It's `iconUrl` :facepalm:

**Comment by alavers** [comment]
I'll merge this, and then cherry pick these changes into old branch that we can publish. :/

**Pull request merged**